sdk/include/reactos/mc/netmsgmsg.mc

```diff
@@ -13038,7 +13038,7 @@ Language=English
 There was an error in creating or reading the alerter mailslot.\nThe error code is %1.
 .
 Language=Polish
-There was an error in creating or reading the alerter mailslot.\nThe error code is %1.
+Wystąpił błąd podczas tworzenia lub odczytywania skrzynki mailslot.\nusługi Alertowanie. Kod błędu to %1.
```

Comment:
`Wystąpił błąd podczas tworzenia lub odczytywania skrzynki mailslot\nurządzenia alarmowego. Kod błędu to %1.`